sources/models/src/modeled_types/kubernetes.rs

```diff
@@ -437,3 +437,120 @@ mod test_kubernetes_bootstrap_token {
         }
     }
 }
+
+// =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+/// EvictionHardKey represents a string that contains a valid Kubernetes eviction hard
+/// signal. There are few valid eviction hard signals [memory.available], [nodefs.available],
```

Comment:
nit: Rather than naming the currently available values, let's just point to the relevant k8s page so if more are added or if they change, we don't need to remember to update this comment.

`Available eviction hard signals: <link here>`